PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 | Q               | A
 | --------------- | ---
-| Bug fix?        | no|yes
-| New feature?    | no|yes
-| BC breaks?      | no|yes
+| Bug fix?        | no\|yes
+| New feature?    | no\|yes
+| BC breaks?      | no\|yes
```

Comment:
I've done this way, as @pamil suggested previously, to keep this ``|`` signs rather than replace them into something else. So I don't know now, should it be ``|`` or ``/`` 😄